engine/src/main/java/org/terasology/input/InputSystem.java

```diff
@@ -284,7 +284,7 @@ public void update(float delta) {
     }
 
     public boolean isCapturingMouse() {
-        return capturingMouse && display.hasFocus();
+        return true;
```

Comment:
Oh this I had to change for the 4th checklist item. It's temporary. The display.hasFocus() method is dependent on the old input system. I'll change it back as soon as I figure out how to change that bit and not have any side effects on the game. :)